packages/design-tokens/src/tailwind/tailwind-presets.ts

```diff
@@ -0,0 +1,126 @@
+// ts-check
+// import {Config} from "tailwindcss"
+import { defaultTheme } from "../"
+
+// TODO: create a color mapper since you can't just pass in destructure colour into here
```

Comment:
This is no longer true ^ will remove this and all other comments in this file later